include/seqan3/contrib/span/span

```diff
@@ -0,0 +1,436 @@
+// -*- C++ -*-
+//!\cond
+//===------------------------------ span ---------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is dual licensed under the MIT and the University of Illinois Open
+// Source Licenses. See LICENSE.TXT for details.
+//
+//===---------------------------------------------------------------------===//
```

Comment:
I assume that this is intended and still applies?

Reply:
Yep, we copied that span implementation from llvm